src/applications/vaos/components/layouts/InPersonLayout.jsx

```diff
@@ -135,8 +135,19 @@ export default function InPersonLayout({ data: appointment }) {
               <FacilityDirectionsLink location={facility} icon />
             </div>
             <br />
-            <span>Clinic: {clinicName || 'Not available'}</span> <br />
-            <span>Location: {clinicPhysicalLocation || 'Not available'}</span>
+            <span>
```

Comment:
This file is missing the masking of type of care above. Also applies to several of the other files in this PR.